src/tracks/Pattern.cpp

```diff
@@ -752,8 +752,6 @@ PatternView::~PatternView()
 
 void PatternView::update()
 {
-	m_pat->changeLength( m_pat->length() );
```

Comment:
This is a bit problematic. If you create a BBTrack with a pattern that goes beyond step 16 ( see #1105/#2964) and you enter this in a song and extend it to cover more than one bar it will fill with one bar sequences. On saving/reloading the project this will now be a sequence of two bar BBTracks, every second bar being empty.